notion-server/src/eth_magic.rs

```diff
@@ -0,0 +1,165 @@
+use std::default;
```

Comment:
For reasoning similar to `EthMagicManager`, I'd call this module just `eth` or `ethereum`.

Reply:
Fair enough -- I'll go with `ethereum`.